providers/ms365/resources/ms365.lr

```diff
@@ -487,6 +487,8 @@ microsoft.policies {
   adminConsentRequestPolicy() dict
   // Permission grant policies
   permissionGrantPolicies() []dict
+  // Consent policy settings
+  consentPolicySettings() dict
```

Comment:
@HRouhani how hard would it be to do this with a dedicated resource instead so we'd have documentation for each field the user could query?

Reply:
It was my initial plan, but not easy as I couldn't find the proper document to see the type of each field, plus it is very dynamic what we get and if we want to add all of them, we need to add over 20 fields.